### ISSUE TITLE
it seems that "ISS pass" part mentioned in readme is not implemented yet

### ISSUE THREAD

**Issue by matkoniecz** [astronomersiva/satellite-passes-notifier#2]
see also http://astronomy.stackexchange.com/questions/14070/notifier-about-visible-iss-passes-and-or-iridium-flares-for-ubuntu-pc/14071#14071

**Comment by astronomersiva** [comment]
Yes, I will push it to the repo this weekend. Have to search it in my old laptop.
